src/views/Orders.vue

```diff
@@ -212,6 +226,25 @@ export default defineComponent({
       });
       return batchmodal.present();
     },
+    async deleteBatch(batch: any) {
```

Comment:
We should have global function that cancels jobs right? Why do we have to define new method here?

Can we not link method directly to action?

Reply:
As we are calling action for delete button of alert.
https://github.com/hotwax/job-manager/pull/140/files#diff-2dfe4d12b346e93d35ad5538343555117ecc5df00c7ad7c067d945535c696d44R241

We could mark it resolved